themes/default/content/blog/finops-with-pulumi/index.md

```diff
@@ -0,0 +1,126 @@
+
+Remediation of any infrastructure mismanagement requires infrastructure reconfiguration action to be taken; often resources need to be replaced with better suited, more cost effective resources,  and resources that were provisioned outside of acceptable policies may have to be destroyed.  However any remediation should carefully consider the underlying reasoning for the misalignment in the first place, both the reason for that initial deployment specifications as well as any decisions for why that’s no longer acceptable.  Be sure to consider all architectural decisions, such as disaster recovery and business continuity planning, regional latency, and any other impact to Service Level Objectives.  Understanding the context will help you code preventative measures into your Pulumi programs going forward and the self-documenting nature of infrastructure-as-code will ensure your decisions are memorialized for future developers and operators, for example, in a Stack README as described.
+
+Automation of Remediation is a goal for many cloud engineering organizations; if the perfect state is defined in code, then any drift may indicate the need to reconcile back to the desired perfect state.  Often, given that some cloud resources may necessarily drift due to limitations in how they are managed, a combination of alerting and remediation of known patterns is implemented.  For example, detecting and deprovisioning any newly instantiated resource not provisioned by Pulumi, but allowing manual resizing of those resources if needed.  Just be sure to capture any infrastructure and improvements you want to make default and/or repeatable as infrastructure as  code.
+
+## The Best First Step: Implement a Tag Policy
+
+Getting started with a FinOps usually falls to the resource owners or Cloud Engineering teams themselves.  The more resources you have, the more complex and noisy things are going to get and the harder it will be to both allocate costs and resources where they are needed, as well as identify and eliminate the unneeded ones.
+
+The best first step to implement is a Tag Policy, the required cloud metadata tags for any resources provisioned by your Organization.  While not every resource in every cloud may be tagged, most are and getting into the practice early will help you maintain accountability for what you are running, demystify your cloud bills as they grow larger, and help with updating the application architecture or making cost effective product decisions long term.
+
+A basic Tag Policy for FinOps will include:
+
+Owner - the person responsible for this resource that you would contact if you had to (eg johndoe@pulumi.com)
```

Comment:
This whole section renders as a single paragraph, which is probably not what you intended. I'd suggest adding blank lines or turning this into a proper bulleted list.